examples/magtag_bitcoin_demo.py

```diff
@@ -33,15 +30,10 @@ def text_transform(val):
 )
 
 magtag.preload_font(b"$012345789")  # preload numbers
```

Comment:
can remove this line, we odnt need to preload for terminalio?

Reply:
Oh right, good catch.